ipyleaflet/leaflet.py

```diff
@@ -5,14 +5,17 @@
 import copy
 import asyncio
 import json
+from folium import LinearColormap
```

Comment:
Should it be imported from `branca`?

Reply:
This importation is a mistake (probably coming from the auto completion of VS code)